package.json

```diff
@@ -89,6 +89,7 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx,d.ts,css,md}": "prettier --write"
+    "*.{js,jsx,ts,tsx,d.ts,css,md}": "prettier --write",
+    "*.{js,jsx,ts,tsx}": "eslint"
```

Comment:
We probably want this as `eslint --fix` to auto fix easy things

Reply:
I've done that in the past and I ended up removing it.

ESLint and Prettier have a different bar when it comes to what is allowed to be autofixed. Prettier will never ever make changes to how your code functions when making autofixes. ESLint plugins will sometimes make unsafe changes in autofixes, and I've seen that cause confusion.

I can definitely change it though if you'd be okay with that risk

Reply:
yeah that's reasonable!